### PR TITLE
LPS-144206

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related Issue: https://issues.liferay.com/browse/LPS-144206

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @rodrigopaulino, can you please check if these failures are related to your changes? Thanks

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_records.jsp [hunk 1]
```diff
@@ -27,10 +27,4 @@ renderResponse.setTitle(LanguageUtil.get(request, "form-entries"));
 	navigationItems="<%= ddmFormViewFormInstanceRecordsDisplayContext.getNavigationItems() %>"
 />
 
-<clay:alert
-	message='<%= LanguageUtil.get(resourceBundle, "view-current-fields-warning-message") %>'
-	style="info"
-	title='<%= LanguageUtil.get(resourceBundle, "info") %>'
-/>
-
```
[the remark this answers is not in the document]
Comment (rodrigopaulino) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_records.jsp:35]: Design's decision. Here's the screen mockup:
Screen Shot 2021-12-23 at 09 50 36

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**